README.md

```
@@ -93,20 +69,13 @@ Apple Developer Academy의 긴 여정 속에서, 러너들은 자신만의 목
 
-이 프로젝트는 [MIT License](LICENSE)를 따릅니다.
-
-
-## 📈 진행 상황
+이 프로젝트는 MIT 라이선스를 따릅니다. 자세한 내용은 [LICENSE](LICENSE) 파일을 참고하십시오.
 
-![Milestone #3](https://img.shields.io/github/milestones/progress/bisor0627/DiverCompass/3)
-![Milestone #4](https://img.shields.io/github/milestones/progress/bisor0627/DiverCompass/4)
-![Milestone #5](https://img.shields.io/github/milestones/progress/bisor0627/DiverCompass/5)
-![Milestone #6](https://img.shields.io/github/milestones/progress/bisor0627/DiverCompass/6)
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)